capellambse/diagram/_diagram.py

```diff
@@ -602,6 +605,7 @@ def __init__(
 
         self.labels = labels or []
         self.hidden = hidden
+        self.context = set(context) if context else set()
 
     def add_context(self, uuid: str) -> None:
         """Do nothing."""
```

Comment:
This needs to be implemented to properly track context.

Reply:
Do I just copy the implementation from Box.add_context?

Reply:
Well, edges can't be ports, so that part doesn't apply at least :)

I'd assume that just the `self.context.add()` call should be enough here.